add information about the chewbacca image (#605)
mattermost/mattermost-developer-documentation: site/content/contribute/containers/_index.md

- [mattermost/podman](https://hub.docker.com/repository/docker/mattermost/podman) - For internal use. Contains Podman to build/tag/push container images.

> - [mattermost/chewbacca](https://hub.docker.com/repository/docker/mattermost/chewbacca-bot) - For internal use. It is a GitHub Bot to help in some administrative tasks for more information you can visit the [Github repo](https://github.com/mattermost/chewbacca).

Reviewer: ```suggestion
- [mattermost/chewbacca](https://hub.docker.com/repository/docker/mattermost/chewbacca-bot) - For internal use. A GitHub Bot for administrative tasks. Please refer to the [GitHub repo](https://github.com/mattermost/chewbacca) for more information.
```

> - [mattermost/matterwick](https://hub.docker.com/repository/docker/mattermost/matterwick) - For internal use. It is a GitHub Bot to spin test servers for Pull requests for more information you can visit the [Github repo](https://github.com/mattermost/matterwick).

Reviewer: ```suggestion
- [mattermost/matterwick](https://hub.docker.com/repository/docker/mattermost/matterwick) - For internal use. A GitHub Bot to spin test servers for pull requests. Please refer to the [GitHub repo](https://github.com/mattermost/matterwick) for more information.
```
